backend/src/models/post.js

```diff
@@ -0,0 +1,10 @@
+import crypto from 'crypto'
+
+export class Post {
```

Comment:
This model does not match the current target schema!
please Update it.
And also add comments for the class.